lib/common/cmdline.c

```diff
@@ -94,7 +96,8 @@ pcmk__build_arg_context(pcmk__common_args_t *common_args, const char *fmts,
             *output_group = g_option_group_new("output", "Output Options:", "Show output help", NULL, NULL);
         }
 
-        output_entries[0].description = crm_strdup_printf("Specify output format as one of: %s", fmts);
+        output_as_descr = crm_strdup_printf("Specify output format as one of: %s", fmts);
+        output_entries[0].description = output_as_descr;
```

Comment:
I'm thinking we should keep output_as_descr in common_args. Then free_common_args() can free it. That makes pcmk__free_arg_context() unnecessary, though we could keep it if we think more will end up there.